### ISSUE TITLE
jsfeat module is not defined

### ISSUE THREAD

**Issue by Omi0** [auduno/clmtrackr#172]
While loading your example I am seeing following error in console:
`Uncaught ReferenceError: module is not defined`
Referencing to jsfeat module. Do you experience the same issue?

**Comment by timbenniks** [comment]
I have this issue when loading clmtrackr through webpack. This error shows when the `start()` function is called.

**Comment by flut1** [comment]
In webpack as a workaround you may want to try loading clmtrackr as an UMD module. Instead of importing from `clmtrackr`, import from `clmtrackr/build/clmtrackr`. Also add to your webpack config:
```
module: {
  noParse: /build\/clmtrackr/,
}
```

**Comment by celestius0** [comment]
This is an issue when trying to load clmtrackr in Angular(8) as well. Any suggestions?

**Comment by ibuioli** [comment]
> This is an issue when trying to load clmtrackr in Angular(8) as well. Any suggestions?

Yes, that is an error from Angular 7+. I made [an implementation](https://github.com/ibuioli/ngFacetracker), but for solve this specific error you need edit the `angular.json` file and add the module like "commonJS" like this:

`"scripts": [ "node_modules/clmtrackr/build/clmtrackr.js" ]`

And then you load it in a TS file (like a component) like this:

`declare var clm: any;`

**Comment by milcktoast** [comment]
Any plans to fix how the module exports are defined?
